.github/workflows/build-and-release-ctl.yml

```diff
@@ -8,12 +8,11 @@ on:
         default: v0.x.x
 
 jobs:
-
```

Comment:
need to add permissions to this:

      id-token: write # needed for signing the images with GitHub OIDC Token